src/library/Uno.Themes.WinUI.Markup/Theme/ToggleButton.cs

```diff
@@ -0,0 +1,334 @@
+						public static ResourceValue<Brush> CheckedDisabled => new("IconToggleButtonBorderBrushCheckedDisabled", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "IconToggleButtonBorderBrushIndeterminate")]
+						public static ResourceValue<Brush> Indeterminate => new("IconToggleButtonBorderBrushIndeterminate", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "IconToggleButtonBorderBrushIndeterminatePointerOver")]
+						public static ResourceValue<Brush> IndeterminatePointerOver => new("IconToggleButtonBorderBrushIndeterminatePointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "IconToggleButtonBorderBrushIndeterminatePressed")]
+						public static ResourceValue<Brush> IndeterminatePressed => new("IconToggleButtonBorderBrushIndeterminatePressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "IconToggleButtonBorderBrushIndeterminateDisabled")]
+						public static ResourceValue<Brush> IndeterminateDisabled => new("IconToggleButtonBorderBrushIndeterminateDisabled", true);
+
+						[ResourceKeyDefinition(typeof(Thickness), "IconToggleButtonBorderThickness")]
+						public static ResourceValue<Thickness> BorderThickness => new("IconToggleButtonBorderThickness", true);
+					}
+
+					public static class StateCircle
+					{
+						public static class Fill
+						{
+							[ResourceKeyDefinition(typeof(Brush), "IconToggleButtonStateCircleFill")]
+							public static ResourceValue<Brush> Default => new("IconToggleButtonStateCircleFill", true);
+						}
+
+						[ResourceKeyDefinition(typeof(double), "IconToggleButtonStateCircleOpacityPointerOver")]
+						public static ResourceValue<double> OpacityPointerOver => new("IconToggleButtonStateCircleOpacityPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(double), "IconToggleButtonStateCircleOpacityPressed")]
+						public static ResourceValue<double> OpacityPressed => new("IconToggleButtonStateCircleOpacityPressed", true);
+
+						[ResourceKeyDefinition(typeof(double), "IconToggleButtonStateCircleOpacityFocused")]
+						public static ResourceValue<double> OpacityFocused => new("IconToggleButtonStateCircleOpacityFocused", true);
+					}
+
+					[ResourceKeyDefinition(typeof(double), "IconToggleButtonMinWidth")]
+					public static ResourceValue<double> MinWidth => new("IconToggleButtonMinWidth", true);
+
+					[ResourceKeyDefinition(typeof(double), "IconToggleButtonMinHeight")]
+					public static ResourceValue<double> MinHeight => new("IconToggleButtonMinHeight", true);
+				}
+			}
+
+			public static class Styles
+			{
+				[ResourceKeyDefinition(typeof(Style), "TextToggleButtonStyle", TargetType = typeof(ToggleButton))]
+				public static ResourceValue<Style> Text => new("TextToggleButtonStyle");
+
+				[ResourceKeyDefinition(typeof(Style), "IconToggleButtonStyle", TargetType = typeof(ToggleButton))]
+				public static ResourceValue<Style> Icon => new("IconToggleButtonStyle");
```

Comment:
Why are these not inside their respective variant?

Reply:
We separate Resources and Styles. So `ToggleButton.Styles` and `ToggleButton.Resources`